example.config.py

```diff
@@ -0,0 +1,6 @@
+token = "YOUR_DISCORD_BOT_TOKEN_HERE" # Obviously you need to specify this.
+groq_api_key = "YOUR_GROQ_OR_OPENAI_API_KEY" # Will disable the AI functions if not provided
```

Comment:
Rename this to ai_api_key or something similar, since it's not groq only anymore